operations/helm/charts/alloy/README.md

```diff
@@ -126,6 +126,12 @@ useful if just using the default DaemonSet isn't sufficient.
 | ingress.tls | list | `[]` |  |
 | nameOverride | string | `nil` | Overrides the chart's name. Used to change the infix in the resource names. |
 | namespaceOverride | string | `nil` | Overrides the chart's namespace. |
+| networkPolicy.egress[0] | object | `{}` |  |
+| networkPolicy.enabled | bool | `false` |  |
+| networkPolicy.flavor | string | `"kubernetes"` |  |
+| networkPolicy.ingress[0] | object | `{}` |  |
+| networkPolicy.policyTypes[0] | string | `"Ingress"` |  |
+| networkPolicy.policyTypes[1] | string | `"Egress"` |  |
```

Comment:
Do we really need such parameters? Why not just write "ingress" if `networkPolicy.ingress` is not nil?

Reply:
Thank you, makes sense! I suppose if helm can distinguish between a nil list and an empty one then we can still do without a `networkPolicy.policyTypes[0]`, but even if that's the case I'm happy with making it more explicit and having the `networkPolicy.policyTypes[0]`.